inspirehep/modules/theme/templates/inspirehep_theme/conferences_in_series.html

```diff
@@ -0,0 +1,27 @@
+{# 
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+#}
+
+<div id="other-conferences-in-the-series">
+  <a href="/conferences/{{record.control_number}}"> {{ record.title }}</a>
+</div>
```

Comment:
I agree with @eamonnmag either we do all rendering server side with Jinja2, or we move everything to JS (Angular).

The second option might bring some extra problems as detailed pages is what we send to Google for indexing, so having everything loaded via Angular has its own set of challenges to index correctly.